### DISCUSSION TITLE
Is it possible to rename ` * ` at TrayTip?

### DISCUSSION THREAD

**Discussion by i448** [spyoungtech/ahk#191] [Q&A] [answered]
I was wondering how to rename the ` Ahk Tray Icon ` so that is easier to know which one is which. Can I do that? Is it possible?

![2023-04-07 21-58-07](https://user-images.githubusercontent.com/128925171/230621382-b602c195-687a-4931-a642-6710c0ce0e0d.gif)

**Comment by spyoungtech** [comment]
I will take this as a feature request. Can you elaborate a bit more on your use-case, specifically how you're starting those processes? Are you using `run_script` by chance?

As of v1 (just released) where the process is daemonized by default. So, if the `Menu, Tray, Tip` command is implemented, that should let you change the tooltip that comes up when you hover over the tray icon. However, that may not work for scripts that are run via `run_script` which would need a different solution.

**Reply by spyoungtech** [reply, marked as answer]
In v1.1.0 new methods are added to support this.

```python
from ahk import  AHK()
ahk = AHK()
ahk.menu_tray_tooltip('My Program Name')
```
You can also change the icon, too:

```python
# change the tray icon (in this case, using a builtin system icon)
ahk.menu_tray_icon('Shell32.dll', 174)

# revert it back to the original:
# ahk.menu_tray_icon()
```
